### PR TITLE
Fix pymedusa/SickRage/issues/704

### PR DESCRIPTION
**Pull request by fernandog**
Fixes https://github.com/pymedusa/SickRage/issues/704

### CODE REVIEW

**Comment by fernandog** [comment]
@ratoaq2 @Thraxis can any of you review this? Im concerned about DB tests in /tests folder

**Comment by fernandog** [comment]
@pymedusa/developers can someone review/test this please?

**Comment by fernandog** [comment]
@ratoaq2 can you review this please?

File: sickbeard/db.py [hunk 1]
```diff
@@ -114,7 +114,17 @@ def _execute(self, query, args=None, fetchall=False, fetchone=False):
...
+            # This errors user should be able to fix it.
+            if 'unable to open database file' in e.args[0] or \
+               'database is locked' in e.args[0] or \
+               'database or disk is full' in e.args[0]:
+                logger.log(u'DB error: {0!r}'.format(e), logger.WARNING)
+            else:
+                logger.log(u'DB error: {0!r}'.format(e), logger.ERROR)
+                raise
         except Exception:
+            logger.log(u'DB error: {0!r}'.format(e), logger.ERROR)
```
Comment (ratoaq2) [inline sickbeard/db.py]: e is not defined here

**Comment by ratoaq2** [comment]
Approved

**Pull request merged**